packages/layout/src/steps/resolvePagination.js

```diff
@@ -19,7 +19,13 @@ const assingChildren = R.assoc('children');
 
 const getHeight = R.path(['box', 'height']);
 
-const allFixed = R.all(isFixed);
+/* All the children are fixed */
+const onlyFixedChildren = n => R.all(isFixed)(n.children);
+
+/* Node itself is fixed OR has only fixed children */
+const isFixedOnly = R.anyPass([isFixed, onlyFixedChildren]);
```

Comment:
Can this function name be `isNodeOrAllChildrenFixed` instead?

Reply:
Updated